miscellaneous/eggnog_performance.md

```diff
@@ -15,4 +15,34 @@ The results are summarized in the table below:
 | 5               | 1                             | N.A. (43 finished)        | 21 min                       |
 | 10              | 1                             | N.A. (39 finished)        | 42 min                       |
 
-Each node has 3 hrs of wall time.
\ No newline at end of file
+(Each node has 3 hrs of wall time.)
+
+### Test with scratch drive (Recommended by NERSC)
+
+Copied the source files to `$SCRATCH` and ran the same tests. The results are summarized in the table below:
+
+| Number of nodes | Parallelization within a node | Time to finish 50 genomes                         | Average node time per genome |
+|-----------------|-------------------------------|---------------------------------------------------|------------------------------|
+| 1               | 1                             | 54.83 mins                                        | 1.1 min                      |
+| 1               | 2                             | 39.77 mins (both batches)                         | 0.79 min                     |
+| 1               | 5                             | 27.92 mins (first batch), 37.11 mins (last batch) | 0.74 min                     |
+| 1               | 10                            | 22.62 mins (first batch), 34.7 mins (last batch)  | 0.69 min                     |
+| 1               | 15                            | 18.8 mins (first batch), 30.79 mins (last batch)  | 0.62 min                     |
+| 2               | 1                             | 25.91 mins (first node), 26.85 mins (last node)   | 1.06 min                     |
+| 5               | 1                             | 10.03 mins (first node), 13.24 mins (last node)   | 1.21 min                     |
```

Comment:
How does this compare to running on the login node w/o scratch?

Reply:
they don't like us to run in login node + we are not going to run our jobs in login node.

Reply:
I'm talking about the data we already have

Reply:
It's hard to say. I think since login nodes are shared, I do notice some performance differences for the same genome set. But I recall with login nodes, average is around 1.5 - 2 mins.

Reply:
But can you put in the times from when you ran it on the login nodes?

Reply:
the job dir got overwritten. I didn't save those info. Login nodes test result is also irrelevant.

Reply:
fair enough

Reply:
:+1: